src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -130,16 +107,9 @@
 						<div v-if="heroBody" class="tw-prose tw-mb-2 md:tw-mb-3">
 							<dynamic-rich-text :html="heroBody" />
 						</div>
-						<kv-button
-							v-if="buttonText"
-							class="tw-w-full md:tw-w-auto"
-							:to="buttonTo"
-							:variant="buttonStyle"
-							@click.native="buttonClick"
-							v-kv-track-event="buttonAnalytics"
-						>
-							{{ buttonText }}
-						</kv-button>
+						<button-wrapper
```

Comment:
Sample use of new component

Reply:
So clean! But what about the responsive classes on the button. Is it behaving that way by default now? Full width on mobile and inline-block on medium and up?